apps/admin-api/env/.env

```
@@ -0,0 +1,3 @@
+CORS_WEBSITE_URLS=https://console-admin.akash.network
+WEBSITE_URL=https://console-admin.akash.network
+LOG_LEVEL=debug
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOG_LEVEL key should go before the WEBSITE_URL key

(UnorderedKey)